### PR TITLE
Add EIP: EVM Object Format (EOFv1) Meta

### PR DESCRIPTION
**Pull request by pdobacz**
Just a list of EOF EIPs.

### CODE REVIEW

File: EIPS/eip-eof-meta.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-eof-meta.md:1]:
```suggestion
---
eip: 7692
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

File: EIPS/eip-eof-meta.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+title: EVM Object Format (EOF) Meta
+description: List of EIPs belonging to the EOF proposal
+author: Paweł Bylica (@chfast), Andrei Maiboroda (@gumb0), Piotr Dobaczewski (@pdobacz), Danno Ferrin (@shemnon)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-evm-object-format-eof-meta/19686
```
Comment (abcoathup) [inline EIPS/eip-eof-meta.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7692-evm-object-format-eof-meta/19686
```

**Review by gumb0 — APPROVED** [review]

**Review by chfast — APPROVED** [review]

**Comment by jochem-brouwer** [comment]
These URLs do not work, please append `.md` to the URLs.

**Comment by gumb0** [comment]
As #8498 is merged now, please also add EIP-7698

**Review by chfast — APPROVED** [review]

**Review by gumb0 — APPROVED** [review]

**Comment by gumb0** [comment]
@g11tech @abcoathup Please take a look.

**Review by g11tech — APPROVED** [review]
lgtm

**Comment by pdobacz** [comment]
> I'm not a fan of these kinds of EIPs. They will get out of date once they go final.
> 
> Unless the intent is to say that once EOF is included, this list will never change and there will never be another EOF proposal? I'd be okay with that.

@SamWilsn 

It has been produced in response to a direct request from ACDE. However, there is a mental shortcut you're pointing out here - it lists EIPs for EOF **v1**, as CFId for Prague. We can somehow make this clear in the text, I'm not exactly sure yet how. Currently it only names it "Mega EOF" in the description, which might not be clear to everyone.

**Review by SamWilsn — APPROVED** [review]

**Review by Usmanali72038 — APPROVED** [review]

**Review by joelythe — COMMENTED** [review]
Y

**Pull request merged**